pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -150,6 +150,9 @@ var (
 		configv1.IBMCloudPlatformType: {
 			iksLBScopeAnnotation: iksLBScopePrivate,
 		},
+		configv1.PowerVSPlatformType: {
+			iksLBScopeAnnotation: iksLBScopePrivate,
+		},
```

Comment:
instead of patching the vendor file which is going to pain always, so can we mimic this variable locally and test this temprorarly? once we have upstream api merges, we can directly submit the PR upstream to use the real one. This is just to avoid all these vendors and rebasing issues!

https://github.com/openshift-powervs/machine-api-operator/blob/master-powervs/pkg/operator/config.go#L19

Reply:
Done